src/pages/Error/NotFound.scss

```
@@ -0,0 +1,30 @@
+.notFoundContainer {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  height: 350px;
+
+  h2 {
+    font-size: 25px;
+    font-weight: 600;
+    margin-bottom: 20px;
+  }
+
+  p {
+    &:last-of-type {
+      margin-bottom: 20px;
+    }
+  }
```

```suggestion
  p:last-of-type {
      margin-bottom: 20px;
  }
```
이렇게 적용하는것과 차이가 없어보입니다!